bucket/mailspring.json

```
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://getmailspring.com/",
+    "description": "Mailspring: The open-source, extensible email app for Mac, Linux, and Windows with open tracking, link click tracking, contacts enrichment data and more.",
+    "license": "GPL-3.0",
+    "version": "1.5.7",
+    "url": "https://github.com/Foundry376/Mailspring/releases/download/1.5.7/Mailspring-1.5.7-full.nupkg#/dl.7z",
```

No need to rename to 7z
```suggestion
    "url": "https://github.com/Foundry376/Mailspring/releases/download/1.5.7/Mailspring-1.5.7-full.nupkg",
```